pkg/client/unversioned/client_test.go

```diff
@@ -38,17 +38,19 @@ import (
 const nameRequiredError = "resource name may not be empty"
 
 type testRequest struct {
-	Method  string
-	Path    string
-	Header  string
-	Query   url.Values
-	Body    runtime.Object
-	RawBody *string
+	Method    string
+	Path      string
+	Header    string
+	Query     url.Values
+	Body      runtime.Object
+	BodyGroup string
```

Comment:
Body is already a runtime.Object; we should be able to figure out the group of a runtime.Object without needing to track it separately.